.github/workflows/ci.yml

```diff
@@ -5,8 +5,8 @@ jobs:
     name: Test
     strategy:
       matrix:
-        python-version: ['3.6.x', '3.7.x', '3.8.x']
-        django-version: ['2.1.15', '2.2.17', '3.0.11', '3.1.4']
+        python-version: ['3.8.x', '3.9.x', '3.10.x', '3.11.x']
+        django-version: ['4.1.7']
```

Comment:
can we test with latest 4.0.x and 4.1.x  ?

Reply:
I thought we want to make sure we support a wide range of python versions and django version just in case some old projects is using that, also PR is not ready, it is still in draft as I have not yet fixed the main issue for translations string not not being picked make messages